web-registry/src/types/rdf/index.ts

```diff
@@ -1,9 +1,14 @@
 export interface URL {
-  '@type': 'http://schema.org/URL';
+  '@type': string;
```

Comment:
probably best to restrict it to:
```suggestion
  '@type': 'http://schema.org/URL' | 'schema:URL';
```

Reply:
Agreed